examples/latex.jl

```diff
@@ -15,13 +15,14 @@ function main(window)
              textinput() >>> texᵗ,
              hbox("Show as a block", hskip(1em), checkbox(mode) >>> modeᵗ),
              vskip(1em),
-             tex(t, block=mode)) |> pad(1em) |> maxwidth(30em)
+             tex(t, block=mode)) |> Escher.pad(1em) |> maxwidth(30em)
     end
 end
+
 function main(window)
```

Comment:
I don't think this is supposed to be there. The first example is much more comprehensive, but because of this definition, it get's overwritten.